### DISCUSSION TITLE
Names in New Portions of Digital Display Gamuts

### DISCUSSION THREAD

**Discussion by cgrobb** [meodai/color-names#137] [General]
Thought the community here would find this interesting. (I have no connection to the company.)

https://6pcolor.s3.amazonaws.com/6PFAQ.pdf

<img width="339" alt="Screen Shot 2022-05-12 at 1 09 15 PM" src="https://user-images.githubusercontent.com/805297/168140827-4393c294-85f9-414f-8522-17411a87ba26.png">

Are the new (to digital display) colors already well-covered with names because of print, nature, etc?

"Because the Yxy signal transport system can support displays with any number of primaries and with the full color spectrum, we see a future with a great variety of displays reproducing colors that viewers have never seen before."

**Comment by meodai** [comment]
they are not covered yet, for now we only cover the standard RGB space. Would be interesting to look into that.

**Reply by meodai** [reply]
@cgrobb do you have a suggestion for that? I'd love to go forward with this. Should we "stretch" the existing colors to fill the bigger gamut? Or should we hand select that would be great fits and provide a p3 value for it?

**Reply by meodai** [reply]
Im willing to go forward with this [cgrobb](https://github.com/cgrobb). what format should we use? xyz d65?

**Reply by cgrobb** [reply]
Apologies for the delayed response. Caught me at end of school year for the kiddos...

I'll follow-up with 6P about common real-world colors that get compromised by gamut reduction in digital and use those colors as a way to inform things.

**Reply by meodai** [reply]
@cgrobb so what format would you maintain colors in?

**Reply by cgrobb** [reply]
https://colorjs.io/docs/spaces.html#xyz-d65 is an obvious choice.

BTW, this from Konica-Minolta has brief discussion of color naming in addition to numerical measurement.

https://www.konicaminolta.com/instruments/knowledge/color/pdf/color_communication.pdf
